test/decorator_test.dart

```diff
@@ -113,6 +113,7 @@ void main() {
 
       group('custom features provided by the consumer', () {
         Feature uniqueConsumerFeature =
+            // ignore: unnecessary_null_comparison
             Feature('canvas', CanvasElement().context2D != null);
```

Comment:
this null-check is no longer needed because `context2D` is non-nullable, but this test wasn't relying on this conditional evaluating one way or another afaict, just the fact that it _has_ a conditional.